src/main/webapp/style.css

```diff
@@ -120,3 +120,10 @@ body {
   position: absolute;
   width: 100%;
 }
+
+.query {
+  font-size: large;
+  font-weight: bold;
+  margin: 3%;
+  padding: 3%;
```

Comment:
I would use fixed values instead of %.

Reply:
I was thinking if I used fixed values then it might not work well on mobile phones and smaller devices

Reply:
SGTM